aiida/orm/implementation/utils.py

```diff
@@ -75,6 +76,11 @@ def clean_builtin(val):
             # see https://www.postgresql.org/docs/current/static/datatype-json.html#JSON-TYPE-MAPPING-TABLE
             raise exceptions.ValidationError('nan and inf/-inf can not be serialized to the database')
 
+        # This fixes an error that occurs when aiida.gaussian passes the output from '_parse_log_cclib'
+        # It contains a datetime.timedelta instance which is not json-serializable
+        if isinstance(val, (datetime.timedelta)):
+            return int(val.total_seconds())
```

Comment:
This is a significant change. I am wondering if the `aiida-gaussian` plugin should simply make sure to remove the datetime, or do the conversion themselves. The reason is that if we do it here, it is a but hidden that the type is changing. If you retrieve the attribute after it has been stored, you won't get the timedelta object that you passed but get an int back.